test/fixtures/output/pipeline/create-with-secret-names.golden

```diff
@@ -1,12 +1,13 @@
-+----------------------+-----------------------------------------------------------------------------------------------------------------------------------+
-| ID                   | pipe-12345                                                                                                                        |
-| Name                 | testPipeline                                                                                                                      |
-| Description          | description                                                                                                                       |
-| KSQL Cluster         | lksqlc-12345                                                                                                                      |
-| Secret Names         | a_really_really_really_really_really_really_really_really_really_really_really_really_long_secret_name_but_not_exceeding_128_yet, |
-|                      | name1, name2, name3                                                                                                               |
-| Activation Privilege | false                                                                                                                             |
-| State                | draft                                                                                                                             |
-| Created At           | 2022-10-04 06:00:00 +0000 UTC                                                                                                     |
-| Updated At           | 2022-10-06 06:00:00 +0000 UTC                                                                                                     |
-+----------------------+-----------------------------------------------------------------------------------------------------------------------------------+
++-------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
+| ID                      | pipe-12345                                                                                                                        |
+| Name                    | testPipeline                                                                                                                      |
+| Description             | description                                                                                                                       |
+| KSQL Cluster            | lksqlc-12345                                                                                                                      |
+| Schema Registry Cluster |                                                                                                                                   |
```

Comment:
None of these tests test the "Schema Registry Cluster" output. Can you modify these tests so we have coverage?

Reply:
Also, you may want to use `omitempty` to hide this row if it's empty?

Reply:
I just noticed that we are not printing SR information in the output so i thought i will add as part of this PR. Let me add few tests to print that.